### ISSUE TITLE
AVT 1 - React FileUploaderButton has a DAP violation

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3316]
## Environment
macOS Mojave version 10.14.5
Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React

## Detailed Description
Run DAP on any of the FileUpLoaderButton and the following DAP violation is present:
<img width="1229" alt="Screen Shot 2019-07-09 at 1 45 04 PM" src="https://user-images.githubusercontent.com/21676914/60914992-5b0a5d00-a250-11e9-81bb-932a7b1bf2fc.png">

Note: To fix this issue remove "role" from the label.
